src/Components/GrowthChart/GrowthChart.tsx

```diff
@@ -30,14 +58,27 @@ export const GrowthChart = () => {
         return [minVal, maxVal];
     }, [min, max, addRangePercentage]);
 
-    const yAxisValues = { minDataValue, maxDataValue };
+    const annotations = GrowthChartAnnotations(xAxisValues, dataSetMetadata?.timeUnit);
 
-    const annotations = GrowthChartAnnotations(xAxisValues, dataSetMetadata.timeUnit);
+    if (isLoading) {
+        return <div>Loading...</div>;
+    }
+
+    if (isError) {
+        console.error('Error fetching tracked entity data.');
+    }
+
+    if (!chartDataForGender || !dataSetValues) {
+        return null;
+    }
```

Comment:
Should be placed as high as possible.

Reply:
Ended up removing isLoading and isError for now, because the Growth Chart component isn't dependent on acquiring a tracked entity. We can discuss adding them back later when we gather growht data from an event.